src/functions/vacation-request/update-vacation-request/handler.ts

```diff
@@ -10,10 +9,10 @@ import type vacationRequestSchema from "src/schema/vacationRequest";
  * @param event event
  */
 const updateVacationRequestHandler: ValidatedEventAPIGatewayProxyEvent<typeof vacationRequestSchema> = async event => {
-  const { pathParameters, body } = event;
```

Comment:
Why not destructure the body here and then use later rather than have the event.body on line 26?